extra-dev/packages/coq-sniper/coq-sniper.dev+8.11+8.12/opam

```diff
@@ -16,12 +16,12 @@ install: [
   [make "install"]
 ]
 depends: [
-  "ocaml" {>= "4.07.1" }
+  "ocaml"
```

Comment:
Let us just remove this dependency completely, as Sniper does not need the OCaml compiler. (MetaCoq and SMTCoq do, but it is the job of their Opam packages to require it.)

But first, let us see if getting rid of three MetaCoq sub-packages get us under the 3h time limit.